test/main_test.ts

```diff
@@ -83,3 +83,11 @@ class A {
     });
   });
 });
+
+describe('regression', () => {
```

Comment:
I think it's fine to leave out the test. It's a corner case internal bug in a dependency - this should be a test in dart-style, but not in ts2dart.